### PR TITLE
13609: Adapt texts and icons.

### PR DESCRIPTION
**Pull request by flxschmidt**
## Description
Adapts texts and icons in exposure detection screens.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13609

## Preview All Changes
image-2022-07-12-15-41-08-265

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -107,22 +107,24 @@
 
 "ExposureDetection_Guide_Hands" = "Waschen Sie Ihre Hände regelmäßig mit Seife für 20 Sekunden.";
 
-"ExposureDetection_Guide_Mask" = "Tragen Sie eine Maske bei Begegnungen mit anderen Personen.";
+"ExposureDetection_Guide_Mask" = "Tragen Sie eine Maske bei Begegnungen mit anderen Personen in Innenräumen und dort, wo es gesetzlich vorgeschrieben ist oder empfohlen wird.";
 
-"ExposureDetection_Guide_Distance" = "Halten Sie mindestens 1,5 Meter Abstand zu anderen Personen.";
+"ExposureDetection_Guide_Distance" = "Halten Sie möglichst mindestens 1,5 Meter Abstand zu anderen Personen. ";
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:112]:
```suggestion
"ExposureDetection_Guide_Distance" = "Halten Sie möglichst mindestens 1,5 Meter Abstand zu anderen Personen.";
```

**Comment by Ein-Tim** [comment]
Can we get a screenshot of the changes?

**Comment by flxschmidt** [comment]
> Can we get a screenshot of the changes?

@30mar could you please provide that when you mock the high risk state? :) thx!!

**Review by naveeddotio — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -107,22 +107,24 @@
...
 
 "ExposureDetection_Guide_Ventilation" = "Lüften Sie Innenräume mehrmals täglich. Öffnen Sie dazu die Fenster einige Minuten lang so weit wie möglich (Stoßlüftung).";
 
-"ExposureDetection_Guide_Symptoms" = "Achten Sie auf Krankheitssymptome. Symptome können auch bei geimpften und bei genesenen Personen auftreten. Falls Sie sich krank fühlen oder Symptome auftreten, wenden Sie sich bitte an Ihre Hausarztpraxis.";
+"ExposureDetection_Guide_Symptoms" = "Achten Sie auf Krankheitssymptome. Symptome können auch bei geimpften und bei genesenen Personen auftreten. Falls Sie sich krank fühlen oder Symptome auftreten, reduzieren Sie Ihre Kontakte, bleiben Sie zu Hause und wenden Sie sich an Ihre Hausarztpraxis.";
 
-"ExposureDetection_Guide_Home" = "Begeben Sie sich, wenn möglich, nach Hause bzw. bleiben Sie zu Hause und vermeiden Sie Begegnungen außerhalb Ihres Haushalts.";
+"ExposureDetection_Guide_ReduceContacts" = "Wenn es Ihnen möglich ist, reduzieren Sie bitte Ihre Kontakte, vor allem mit besonders gefährdeten Personen, z.B. Menschen hohen Alters, mit Vorerkrankungen oder einem geschwächten Immunsystem";
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:120]:
```suggestion
"ExposureDetection_Guide_ReduceContacts" = "Wenn es Ihnen möglich ist, reduzieren Sie bitte Ihre Kontakte, vor allem mit besonders gefährdeten Personen, z.B. Menschen hohen Alters, mit Vorerkrankungen oder einem geschwächten Immunsystem.";
```

File: src/xcode/ENA/ENA/Resources/Localization/bg.lproj/Localizable.strings [hunk 3]
```diff
@@ -116,8 +116,6 @@
 
 "ExposureDetection_Guide_Symptoms" = "Следете за типичните симптоми на COVID-19. Дори ваксинираните хора и преболедувалите може да проявят симптоми. Ако не се чувствате добре или проявявате симптоми моля, свържете се с Вашия личен лекар.";
 
-"ExposureDetection_Guide_Home" = "По възможност се приберете и си останете вкъщи, избягвайки срещи с хора, които не са от Вашето домакинство.";
```
[the remark this answers is not in the document]
Comment (flxschmidt) [reply src/xcode/ENA/ENA/Resources/Localization/bg.lproj/Localizable.strings:119]: ah, that's good to know!

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 2 above)
Comment (flxschmidt) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:120]: yes in Figma the dot is missing as well. Good catch!

**Review by 30mar — CHANGES_REQUESTED** [review]
the info button should be top aligned acording to the design in the High risk
Simulator Screen Shot - iPhone 13 - 2022-07-12 at 16 27 17

**Review by 30mar — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift [hunk 4]
```diff
@@ -497,7 +497,17 @@ class ExposureDetectionViewModel: CountdownTimerDelegate {
 								link: AppStrings.Links.exposureDetectionFAQ
 							)
 						]),
-					.guide(text: AppStrings.ExposureDetection.guideVaccinationHighRisk, image: UIImage(named: "Icons - VaccinatedArm")),
+					.guide(
+						titleText: NSAttributedString(string: AppStrings.ExposureDetection.guideVaccinationHighRisk),
+						titleImage: UIImage(named: "Icons - VaccinatedArm"),
+						linkedTexts: [
+							LinkedText(
+								text: AppStrings.ExposureDetection.guideVaccinationHighRisk_LinkText,
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift:505]:
```suggestion
								text: AppStrings.ExposureDetection.guideVaccinationHighRisk,
```

File: src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift [hunk 5]
```diff
@@ -497,7 +497,17 @@ class ExposureDetectionViewModel: CountdownTimerDelegate {
 								link: AppStrings.Links.exposureDetectionFAQ
 							)
 						]),
-					.guide(text: AppStrings.ExposureDetection.guideVaccinationHighRisk, image: UIImage(named: "Icons - VaccinatedArm")),
+					.guide(
+						titleText: NSAttributedString(string: AppStrings.ExposureDetection.guideVaccinationHighRisk),
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift:501]:
```suggestion
```

**Review by OLVI5 — APPROVED** [review]

**Pull request merged**